addons/website_sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.xml

```diff
@@ -13,27 +13,59 @@
         <button name="sale_combo_configurator_cancel_button" position="attributes">
             <attribute name="t-if">!this.props.isFrontend</attribute>
         </button>
+        <div name="sale_combo_configurator_quantity_wrapper" position="attributes">
+            <attribute name="t-if">!this.props.isFrontend</attribute>
+        </div>
+        <div name="sale_combo_configurator_total_wrapper" position="attributes">
+            <attribute name="t-if">!this.props.isFrontend</attribute>
+        </div>
         <button name="sale_combo_configurator_confirm_button" position="after">
-            <button
-                name="website_sale_combo_configurator_continue_button"
-                class="btn btn-secondary flex-grow-1 flex-md-grow-0"
-                t-if="this.props.isFrontend and !this.props.options.isBuyNow"
-                t-on-click="() => this.confirm({goToCart: false})"
-                t-att-disabled="!areAllCombosSelected"
-            >
-                Continue Shopping
-            </button>
-        </button>
-        <button name="sale_combo_configurator_cancel_button" position="after">
-            <button
-                name="website_sale_combo_configurator_checkout_button"
-                class="btn btn-primary flex-grow-1 flex-md-grow-0"
-                t-if="this.props.isFrontend"
-                t-on-click="() => this.confirm({goToCart: true})"
-                t-att-disabled="!areAllCombosSelected"
-            >
-                Proceed to Checkout
-            </button>
+            <div class="d-flex flex-column-reverse flex-lg-row gap-3 w-100">
+                <div class="d-flex flex-column flex-md-row gap-2">
+                    <button
+                        name="website_sale_combo_configurator_continue_button"
+                        class="btn btn-primary w-100 w-lg-auto"
+                        t-if="this.props.isFrontend and !this.props.options.isBuyNow"
+                        t-on-click="() => this.confirm({goToCart: false})"
+                        t-att-disabled="!areAllCombosSelected"
+                    >
+                        <i class="fa fa-shopping-cart me-2"/>
+                        Add to Cart
+                    </button>
+                    <button
+                        name="website_sale_combo_configurator_checkout_button"
+                        class="btn btn-outline-primary w-100 w-lg-auto"
+                        t-if="this.props.isFrontend"
+                        t-on-click="() => this.confirm({goToCart: true})"
+                        t-att-disabled="!areAllCombosSelected"
+                    >
+                        Go to Checkout
```

Comment:
In the product configurator we still use `Proceed to Checkout` to stay consistent I'd suggest renaming accordingly.